ui.apps/src/main/content/jcr_root/apps/saas-aem-module/clientlibs/wknd-sample/css/styles.less

```diff
@@ -1,376 +0,0 @@
-.search * {
```

Comment:
I am a bit confused. Why did you had to change this? was it already wrong before? cc @kevinolivar

Reply:
The old styling is obsolete (the one you started with) as now we have the clientlib generating the wknd-sample (styling "demo") as follow,


````
{
      ...libsBaseConfig,
      name: 'wknd-sample',
      categories: ['saas-aem-module.wknd.sample'],
      dependencies: ['saas-aem-module.dependencies'],
      assets: {
        // Copy entrypoint stylesheets into the respective ClientLib
        // directories
        css: {
          cwd: 'clientlib-base',
          files: ['**/*.css'],
          flatten: false,
        },

        // Copy all other files into the `resources` ClientLib directory
        resources: {
          cwd: 'clientlib-base',
          files: ['**/*.*'],
          flatten: false,
          ignore: ['**/*.js', '**/*.css'],
        },
}
`````